requests/signed_message_test.go

```diff
@@ -80,7 +80,7 @@ func TestRequestShouldBuildForValid(t *testing.T) {
 	assert.Check(t, urlCheck)
 	assert.Check(t, signed.Header.Get("X-Yoti-Auth-Digest") != "")
 	assert.Equal(t, signed.Header.Get("X-Yoti-SDK"), "Go")
-	assert.Equal(t, signed.Header.Get("X-Yoti-SDK-Version"), "3.9.0")
+	assert.Equal(t, signed.Header.Get("X-Yoti-SDK-Version"), "3.11.0")
```

Comment:
since this file was copy pasted into `digitalidentity/` package, that one also has this equivalent test file that needs to be updated